src/components/space/charts-layout/ChartsHeader.js

```diff
@@ -47,10 +47,9 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function ChartsHeader({ downloadButton }) {
-  const { t } = useTranslation();
+const ChartsHeader = ({ downloadButton }) => {
   const classes = useStyles();
-  const { spaceName } = useContext(SpaceDataContext);
+  const { spaceName } = useContext(ComposeDataContext);
```

Comment:
Why do you use `ComposeDataContext` here?

Reply:
Previously, there was one context/provider for the actions API call, called `SpaceDataProvider`. With this update, it would have been unwieldy to do both API calls in one context, so I split the actions API calls into two contexts/providers. I had to choose one of them from which to extract the spaceName (which is the same, since both return the same spaceTree), and I chose the default one (the _Compose_ view).